src/cms-content/articles/index.ts

```diff
@@ -16,20 +20,28 @@ interface ArticleJSON extends ArticleMain {
 }
 
 export interface Article extends ArticleMain {
-  date: Date;
+  date: string;
 }
 
 function sanitizeArticle(article: ArticleJSON): Article {
   return {
     ...article,
     articleID: sanitizeID(article.articleID),
-    date: moment(article.date).toDate(),
+    date: moment(article.date).format('MMM Do, YYYY'),
   };
 }
 
 const articleList: ArticleJSON[] = [
-  // indigenousPeoplesDay
+  thirdSurge,
+  indigenousPeoplesDay,
+  animap,
+  canCompare,
 ];
 
 const articles: Article[] = articleList.map(sanitizeArticle);
+
```

Comment:
Consider sorting the articles by date (most recent first) so they always show in that order in the landing page